invadedhouse/commands/inventory.py

```diff
@@ -0,0 +1,29 @@
+from commands import Command
+
+
+class Inventory(Command):
+
+    def do(self, text: str):
+        player = self.get_player()
+
+        print('Inventory:')
+        if not player.all_items():
+            print('Nothing.')
+        for item in player.all_items():
+            print(item)
+
+    def get_description(self):
+        return 'In case you forget what you\'re holding.'
+
+    def get_aliases(self):
+        return [
+            'show inventory',
+            'what am i holding',
+            'what do i have',
+            'my stuff',
+            'my items',
+            'i',
```

Comment:
this says 'i', in minecraft its 'e'. But it wouldnt be a good idea to add 'e' to this if it has 'i'.

Reply:
minecraft did that btw so that the inventory button was super easy to reach from the wasd walk commands (left hand keyboard right hand mouse)